ratpack-bearer-auth/src/main/java/st/ratpack/auth/springsec/SpringSecCheckAuthModule.java

```diff
@@ -12,12 +12,11 @@
 
 	@Override
 	protected void configure() {
-	}
+		OptionalBinder.newOptionalBinder(binder(), TokenProvider.class)
+			.setDefault().to(SpringSecCheckTokenProvider.class).in(Scopes.SINGLETON);
```

Comment:
Yeah, I saw that as well.  If you're doing something expensive in a default, you still need to feature it flag it with the module configs.